audits/performance/first-meaningful-paint.js

```diff
@@ -28,25 +28,32 @@ class FirstMeaningfulPaint {
     return 'Has a good first meaningful paint';
   }
 
+  /**
+   * Audits the page to give a score for First Meaningful Paint.
+   * @see  https://github.com/GoogleChrome/lighthouse/issues/26
+   * @param  {Object} inputs The inputs from the gather phase.
```

Comment:
I would prefer strict typing across everything.

closure annotations would be our best bet since it required no transplantation, and we get compile time checks.